tests/subsys/debug/coredump_backends/testcase.yaml

```diff
@@ -16,7 +16,12 @@ tests:
   coredump.backends.flash:
     filter: CONFIG_ARCH_SUPPORTS_COREDUMP
     extra_args: CONF_FILE=prj_flash_partition.conf
-    platform_allow: qemu_x86
+    platform_allow:
+      - qemu_x86
+      - esp32
+      - esp32s2_saola
+      - esp32s3_devkitm
+      - esp32c3_devkitm
```

Comment:
Is coredump for `esp32c3_devkitm`  fully working?

Reply:
Both logging and flash backend